web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/forvaltning/ForvaltningBeregningRestTjeneste.java

```diff
@@ -205,6 +205,35 @@ private Behandling getBehandling(ForvaltningBehandlingIdDto dto) {
         return behandlingRepository.hentBehandling(dto.getBehandlingUuid());
     }
 
+    @POST
+    @Path("/stoppRefusjon")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(description = "Setter opphørsdato for refusjon for en gitt journalpost", tags = "FORVALTNING-beregning")
+    @BeskyttetRessurs(actionType = ActionType.READ, resourceType = ResourceType.DRIFT, sporingslogg = false)
+    public Response opphørRefusjonInntektsmelding(@BeanParam @Valid StoppRefusjonDto dto) {
+        var behandling = behandlingRepository.hentBehandling(dto.getBehandlingUuid());
+        var inntektsmeldinger = iayTjeneste.finnGrunnlag(behandling.getId())
+            .flatMap(InntektArbeidYtelseGrunnlag::getInntektsmeldinger)
+            .map(InntektsmeldingAggregat::getAlleInntektsmeldinger)
+            .orElse(Collections.emptyList());
+        var matchetInntektsmelding = inntektsmeldinger.stream().filter(im -> im.getJournalpostId().getVerdi().equals(dto.getJournalpostId())).findFirst();
+        if (matchetInntektsmelding.isEmpty()) {
+            var msg = String.format("Finner ikke inntektsmelding med journalpostId %s på behandling med uuid %s ", dto.getJournalpostId(),
+                dto.getBehandlingUuid());
+            return Response.ok(msg).build();
+        }
+        var task = ProsessTaskData.forProsessTask(OverstyrInntektsmeldingTask.class);
+        task.setBehandling(behandling.getFagsakId(), behandling.getId(), behandling.getAktørId().getId());
+        task.setCallIdFraEksisterende();
+        task.setProperty(OverstyrInntektsmeldingTask.BEHANDLING_ID, behandling.getId().toString());
+        task.setProperty(OverstyrInntektsmeldingTask.JOURNALPOST_ID, dto.getJournalpostId());
+        task.setProperty(OverstyrInntektsmeldingTask.OPPHØR_FOM, dto.getRefusjonOpphørFom().toString());
+        task.setProperty(OverstyrInntektsmeldingTask.SAKSBEHANDLER_IDENT, KontekstHolder.getKontekst().getKonsumentId());
```

Comment:
@mrsladek blir dette riktig for å få med oss ident på saksbehandler?

Reply:
Tror da får du Id'en til selve appen som kaller. Du bør bruke `getUid()`

Reply:
Endrer til det